src/redux/selectors/pagination/getPagerCurrentlyShown.js

```diff
@@ -0,0 +1,9 @@
+export default function getPagerCurrentlyShown(state) {
+    const { pageSize, total, pageCount, page } = state.pagination
+    const pageCalculationValue =
+        total - (total - (pageCount - (pageCount - page)) * pageSize)
```

Comment:
> `pageCount - (pageCount - page)`

When removing the brackets, this turns into
=> `pageCount - pageCount + page`.

So it's equivalent to
=> `page`

So it looks like this:
=> `total - (total - page * pageSize)`

I'll substitute `page*pageSize` with `pagePageSize`
=> `total - (total - pagePageSize)`

When I remove the bracket, it turns into:
=> `total - total + pagePageSize`

Which is equivalent to:
`pagePageSize`

So the whole line can be changed to:
`const pageCalculationValue = page * pageSize`